actors/user/src/main/java/org/sunbird/user/util/UserUtil.java

```diff
@@ -626,6 +626,29 @@ public static void validateUserExternalIds(User user) {
     return organisations;
   }
 
+  @SuppressWarnings("unchecked")
+  public static List<Map<String, Object>> getAllUserOrgDetails(String userId) {
```

Comment:
don't repeat the code - 

original method - nam to getActiveuserOrgDetails
this method name is ok - make the code common